src/main/kotlin/coffeebot/commands/Milton.kt

```diff
@@ -0,0 +1,58 @@
+package coffeebot.commands
+
+import coffeebot.message.Valid
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.okhttp.OkHttp
+import io.ktor.client.features.HttpTimeout
+import io.ktor.client.features.json.GsonSerializer
+import io.ktor.client.features.json.JsonFeature
+import io.ktor.client.request.post
+import io.ktor.client.statement.HttpResponse
+import io.ktor.http.HttpStatusCode
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
+import java.lang.IllegalStateException
+import java.net.MalformedURLException
+import java.net.URL
+
+class MiltonClient {
+    private val httpClient = HttpClient(OkHttp) {
+        install(JsonFeature) {
+            serializer = GsonSerializer()
+        }
+        install(HttpTimeout) {
+            requestTimeoutMillis = 45_000
+            socketTimeoutMillis = 45_000
```

Comment:
45 secs seems a bit high

Reply:
Unless MIlton's cold start takes that long?

Reply:
Cold starts do take very long, but I think 25 seconds would be enough. I'll change that.